docs/management_api.yml

```diff
@@ -467,7 +464,7 @@ paths:
       security:
         - ManagementJWT: []
       summary: |
-        Get list of all devices and their respective status for the deployment
+        Get list of devices based on a status query for the deployment
         with the given ID.
```

Comment:
I'd suggest something like this:

Get the list of devices and their respective status for the deployment
with the given ID. The response includes devices as they get assigned to the deployment when checking for updates. Therefore, this endpoint will list all the devices only once each asks for updates and evaluates this deployment.

Reply:
To me it sounds very confusing...

Reply:
confusing or not this is exactly how this endpoint is working

Reply:
Makes sense. Thanks 👍